packages/pipeline/src/scripts/pull_competing_dex_trades.ts

```diff
@@ -39,14 +44,29 @@ async function getAndSaveTradesAsync(): Promise<void> {
 }
 
 async function getLastSeenTimestampAsync(tradesRepository: Repository<DexTrade>): Promise<number> {
+    const hoursPerDay = 24;
+    const minutesPerHour = 60;
+    const secondsPerMinute = 60;
+    const millisecondsPerSecond = 1000;
+    const millisecondsPerDay = millisecondsPerSecond * secondsPerMinute * minutesPerHour * hoursPerDay;
```

Comment:
IMO we don't need to be quite so verbose here in declaring the different units.

```
const millisecondsPerDay = 1000 * 60 * 60 * 24;
```

Seems clear enough. But this is also subjective.